CHANGELOG.md

```diff
@@ -2,6 +2,27 @@
 
 All notable changes to this project will be documented in this file. See [standard-version](https://github.com/conventional-changelog/standard-version) for commit guidelines.
 
+## [0.33.0](https://github.com/aws/aws-rfdk/compare/v0.32.0...v0.33.0) (2021-06-01)
+
+
+### Supported CDK Version
+
+* [1.106.1](https://github.com/aws/aws-cdk/releases/tag/v1.106.1)
+
+
+### Officially Supported Deadline Versions
+
+* [10.1.9.2 to 10.1.15.2](https://docs.thinkboxsoftware.com/products/deadline/10.1/1_User%20Manual/manual/release-notes.html)
+
+
+### Bug Fixes
+
+* **core:** improve security of mongodb setup script ([#445](https://github.com/aws/aws-rfdk/issues/445)) ([9148f25](https://github.com/aws/aws-rfdk/commit/9148f25c9864b6f09d53065b4ff4be74299f40f1))
+* **deadline:** allow zero-sized WorkerInstanceFleet ([#451](https://github.com/aws/aws-rfdk/issues/451)) ([0cc6723](https://github.com/aws/aws-rfdk/commit/0cc67238645e24805590412776d262c8e6b9ec49))
+* **deadline:** use IMDSv2 endpoint in Repository scripting ([#436](https://github.com/aws/aws-rfdk/issues/436)) ([e7eddea](https://github.com/aws/aws-rfdk/commit/e7eddea0c37b143b2eed7db7ed62a42960cac95e))
+* **integ:** fix broken CDK app deny-list and error handling ([#435](https://github.com/aws/aws-rfdk/issues/435)) ([cfcb9bb](https://github.com/aws/aws-rfdk/commit/cfcb9bba99ec60a60cff2939f454be38a6795f28))
+* **integ:** make exit code reflect test success/failure ([#456](https://github.com/aws/aws-rfdk/issues/456)) ([ff40a93](https://github.com/aws/aws-rfdk/commit/ff40a930cbf6f3e7d46328291ae239dbb9fe011d))
```

Comment:
We probably don't need these published in the release notes - they're purely developer-facing.

Reply:
Good catch, removed